examples/app/Embeddings.ipynb

```diff
@@ -20,36 +48,257 @@
+    }
+   },
+   "cell_type": "markdown",
+   "id": "7619577f",
+   "metadata": {},
+   "source": [
+    "You can change the type of the displayed weave panel by changing `Table` to `Plot` and it will intelligently change to display a plot of the data.\n",
+    "![image.png](attachment:image.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "356dc2a9",
+   "metadata": {},
+   "source": [
+    "## Embed and Cluster the data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4fd47c57",
+   "metadata": {},
+   "source": [
+    "Now we can pass a column from this data to another weave `Op` to embed it. This will turn each row in the column `name` into a vector."
```

Comment:
minor: would love a brief note on the intuition here—what is vectorizing Pokemon names going to give us/why might we want to do this? Would we expect similar-sounding parts of names to end up close together...?

Reply:
yeah, or maybe openai has some internal knowledge of Pokemon. But I think we update this to have better data.